dali/operators/image/crop/crop_attr.cc

```diff
@@ -87,41 +85,47 @@ CropAttr::CropAttr(const OpSpec& spec) {
                  "`crop_d` argument must be provided together with `crop_w` and `crop_h`");
   }
 
-  size_t crop_arg_ndims = 0;
+  int crop_d = kNoCrop, crop_h = kNoCrop, crop_w = kNoCrop;
```

Comment:
`k` usually meant "constant", which these are `int`, not `const int`. Is that intentional?

Reply:
it's just initialized to kNoCrop. Replaced later if applicable.
kNoCrop is constexpr:
```static constexpr int kNoCrop = -1;```